src/expr/src/scalar/mod.rs

```diff
@@ -1056,6 +1057,9 @@ impl fmt::Display for EvalError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             EvalError::DivisionByZero => f.write_str("division by zero"),
+            EvalError::FeatureNotSupported(feature) => {
+                write!(f, "feature not supported: {}", feature)
```

Comment:
Including "feature not supported" in the error text here seems maybe a bit weird. Doesn't seem to add any clarity to the error messages you've got so far, at least!

Reply:
I would love if we had something Cockroach does where the error message actually links to a GitHub issue tracking support for that family of functions, [like here](https://github.com/cockroachdb/cockroach/blob/2e326b85fd0d60a62ae9d344b0be5c89e36bcc30/pkg/sql/sem/builtins/builtins.go#L3626).